backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/nuernberg/constants/Categories.kt

```diff
@@ -0,0 +1,15 @@
+package app.ehrenamtskarte.backend.stores.importer.nuernberg.constants
+
+val categoryMapping =
```

Comment:
just kept the mapping with default to ensure that wrong categoryIds will mapped to defaultValue `9`